t/50reverse-proxy-http2-trailers.t

```diff
@@ -12,12 +12,11 @@ subtest 'nghttp2 client and backend' => sub {
         unless prog_exists('nghttpd');
 
     my ($backend_port) = empty_ports(1, { host => '0.0.0.0' });
-    my $backend = spawn_server(
-        argv => ['nghttpd', '--htdocs', DOC_ROOT,
-                 '--trailer', 'x-backend-trailer: bar',
-                 $backend_port, 'examples/h2o/server.key', 'examples/h2o/server.crt'],
-        is_ready => sub { check_port($backend_port) },
-    );
+    my $backend = spawn_forked(sub {
+        exec('nghttpd', '-v', '--htdocs', DOC_ROOT,
+             '--trailer', 'x-backend-trailer: bar',
+             $backend_port, 'examples/h2o/server.key', 'examples/h2o/server.crt');
+    });
```

Comment:
Is there a reason to make this change?

I think we might prefer using `spawn_server` as it takes care of launching a sub-process and also has the capability of speciying a wait function (i.e., `is_ready`).

Reply:
It's just to read stdout in a convenient manner.

Reply:
Yeah you are right, the TODO test that's being commented out relies on that. I'm fine with keeping the changee as-is, modulo my other comment.